src/unity/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -342,7 +362,14 @@ def _load_version(cls, state, version):
         # Load the custom nerual network
         num_classes = state['num_classes']
         num_inputs = state['_feature_extractor'].output_length
-        net = SoundClassifier._build_custom_neural_network(num_inputs, num_classes)
+        if 'custom_layer_sizes' in state:
+            # These are deserialized as floats
+            custom_layer_sizes = map(int, state['custom_layer_sizes'])
```

Comment:
Do we want to make it `list(map(...))` to not break on Python 3?

Reply:
Thanks. Done.